src/schema/mutation/editApiConfiguration.mutation.ts

```diff
@@ -77,6 +78,8 @@ export default {
         },
         args.permissions && { permissions: args.permissions }
       );
+
+      console.log(update);
```

Comment:
@RenzoPrats 
ℹ️ read again your code after pushing stuff
you should remove the logs